utils/src/ooinstall/openshift_ansible.py

```diff
@@ -127,7 +127,7 @@ def default_facts(hosts, verbose=False):
 
 def run_main_playbook(hosts, hosts_to_run_on, verbose=False):
     global CFG
-    inventory_file = generate_inventory(hosts)
+    inventory_file = generate_inventory(hosts_to_run_on)
```

Comment:
That said, we should create a group in the inventory for hosts_to_run_on, and that group can be the one referenced in the scaleup playbook.